packages/backend/src/index.mjs

```diff
@@ -120,3 +120,9 @@ for (const routeFile of routes) {
 //     error: JSON.stringify(error, null, 2),
 //   })
 // })
+
+export const clearCollection = async (collection, options) => {
+  if (await db.findOne(collection, options)) {
+    await db.delete(collection, options)
+  }
+}
```

Comment:
I think this can be in test 😅

Reply:
e.g. `test/utils.mjs` or just put in `attesterDescription.mjs`

Reply:
yep fixed